client/src/components/meet-the-team/MeetTheTeam.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: "League Spartan";
+}
+
+h1 {
+  font-size: 4rem;
+  margin-bottom: 1rem;
+}
+
+h2 {
+  margin-top: 0.8rem;
+  font-size: 2.3rem;
+}
+
+
```

Comment:
maybe blank line 15 could be removed

Reply:
Thanks Maria! I'll add these to do when I've merged :) I think the footer issue might be a global CSS thing so will check it works across other pages too.